account_financial_report_webkit_xls/report/partners_balance_xls.py

```diff
@@ -376,7 +376,7 @@ def generate_xls_report(self, _p, _xs, data, objects, wb):
                              regular_cell_style_decimal),
                         ]
                     else:
-                        debit_col = 3
+                        debit_col = 4
```

Comment:
It must be so because when the initial balance column is not exported, the account / Partner name column takes columns a, b and c and the formula takes the data from the Code / Ref column.